sources/packages/web/src/views/aest/student/StudentRestrictions.vue

```diff
@@ -6,14 +6,18 @@
       :canResolveRestriction="true"
     ></student-restrictions
   ></tab-container>
+  <tab-container>
```

Comment:
The tab container must be only one per view. I see that the scholastic standing limited history will go under same card. But I am not sure if we need to be very specific about the UI.

But that is so, we can achieve the same look and feel by using the property `enableCardView` in tab container and body header container to achieve it.

@andrewsignori-aot @guru-aot  @andrepestana-aot  thoughts?

Reply:
I agree that the `tab-container` is intended to be the only child in a tab. IMO we should follow @dheepak-aot recommendation on this 😉